src/test/java/com/ays/user/service/impl/UserSelfServiceImplTest.java

```diff
@@ -35,6 +37,65 @@ class UserSelfServiceImplTest extends AbstractUnitTest {
     @Mock
     private AysIdentity identity;
 
+
+    private final UserEntityToUserMapper userEntityToUserMapper = UserEntityToUserMapper.initialize();
+
+
+    @Test
+    void whenUserFound_thenReturnUser() {
+        // Given
+        UserEntity mockUserEntity = new UserEntityBuilder()
+                .withValidFields()
+                .build();
+        String mockUserId = mockUserEntity.getId();
+
+        User mockUser = userEntityToUserMapper.map(mockUserEntity);
+
+        // When
+        Mockito.when(identity.getUserId())
+                .thenReturn(mockUserId);
+        Mockito.when(userRepository.findById(Mockito.anyString()))
+                .thenReturn(Optional.of(mockUserEntity));
+
+        // Then
+        User user = userSelfService.getUserSelfInformation();
+
+        Assertions.assertEquals(mockUser.getId(), user.getId());
+        Assertions.assertEquals(mockUser.getUsername(), user.getUsername());
+
+        Mockito.verify(identity, Mockito.times(1))
+                .getUserId();
+        Mockito.verify(userRepository, Mockito.times(1))
+                .findById(mockUserId);
+    }
+
+    @Test
+    void whenUserIsNotExistForGetting_thenThrowAysUserNotExistByIdException() {
+        // Given
+        UserEntity mockUserEntity = new UserEntityBuilder()
+                .withValidFields()
+                .build();
+        String mockUserId = mockUserEntity.getId();
```

Comment:
Bu kısımda mockUserEntity başka birşey için kullanılmıyor. `AysRandomUtil.generateUUID()` fonksiyonunu kullanırsak mockUserEntity'e ihtiyaç kalmıyacaktır

Reply:
Fixed ✅